oauth2client/contrib/sql_alchemy.py

```diff
@@ -0,0 +1,86 @@
+"""OAuth 2.0 utilities for SQLAlchemy.
+Utilities for using OAuth 2.0 in conjunction with SQLAlchemy.
+Heavily inspired by equivalent Django module.
+"""
+
+from oauth2client.client import Storage as BaseStorage
+from sqlalchemy.types import PickleType
+
+
+__author__ = 'dominik@mdzn.pl (Dominik Miedzinski)'
+
+
+class CredentialsType(PickleType):
+    pass
+
+
+class FlowType(PickleType):
+    pass
+
+
+class Storage(BaseStorage):
+    """Store and retrieve a single credential to and from the SQLAlchemy.
```

Comment:
I'll keep it in mind, but no worries. Even those of us who are native speakers need proofreaders. :)

Keep in mind that I do hold documentation to a very high standard, so please try not to get too frustrated with me - and let me know if I'm asking too much of you.